tests/Doctrine/Tests/DBAL/Types/TypeTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\DBAL\Types;
+
+use Doctrine\DBAL\Types\Type;
+use PHPUnit\Framework\TestCase;
+use ReflectionClass;
+
+class TypeTest extends TestCase
+{
+    /**
+     * @dataProvider defaultTypesProvider()
```

Comment:
Drop the parenthesis, they are not used in any other tests except `StatementIteratorTest`. Drop them there too.

Reply:
@morozov WDYT? Didn't we agree to prefer them?

Reply:
Personally, I like it more with parentheses since this is how functions and methods are usually notated in PHP documentation. It'd be nice to have it clarified in the standard.